### ISSUE TITLE
Initial Bearing Calculation Error

### ISSUE THREAD

**Issue by rickogden** [php-geospatial/geospatial#26]
I've been getting incorrect results from the initial bearing calculation. I created a phpt file which shows the issue.
[initial_bearing2.zip](https://github.com/php-geospatial/geospatial/files/2736378/initial_bearing2.zip)

**Comment by derickr** [comment]
I'm just copying the code in here so that I don't have to download and unzip a file every time:

```
<?php
$from = array(
    'type' => 'Point',
    'coordinates' => array( 2.351, 48.857 )
);
$to = array(
    'type' => 'Point',
    'coordinates' => array( 0.119, 52.205 )
);
var_dump(initial_bearing($from, $to));
?>
```

Curious. The result is an exact complement to 360° as what you expected:

```
001+ float(22.1095598095075)
001- float(337.89044019049)
```

**Comment by derickr** [comment]
I've verified that your expected answer is correct with http://www.gcmap.com/mapui?P=48.857%2C2.351-52.205%2C0.119&MS=wls&DU=mi (click on "Show fractional heading")

**Comment by rickogden** [comment]
Yeah, it seems like some inversion is going on. My calculations and also movable-type agree that it should be 337.

**Comment by rickogden** [comment]
https://github.com/rickogden/Location/blob/master/src/Location.php#L462 Not sure if it's any help, but this PHP method returns the correct result.

**Comment by derickr** [comment]
I see the problem. Would you be OK if it returned -22°? Its range would be -180° to +180° instead of 0° to +360°C?

**Comment by rickogden** [comment]
I can certainly work with that, but it seems a bit non-standard.

**Comment by derickr** [comment]
Reading up on this, I think that's not wise. I'll normalise to 0° to +360°.

**Comment by derickr** [comment]
Merged, and released through 0.2.1.
